code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -26,6 +26,14 @@
 	mag_type = /obj/item/ammo_box/magazine/m45
 	can_suppress = 0
 
+/obj/item/weapon/gun/projectile/automatic/pistol/empty	//empty stetchshit for maint spawns
+	desc = "A small, easily concealable 10mm handgun. Had a threaded barrel for suppressors."
+	can_suppress = 0

```

Comment:
Felt too powerful for me while making it, I can change that, though, to make it identical to the regular stetchkin.